cli/cmd/resources.go

```diff
@@ -286,6 +259,31 @@ var (
 		// deprecated resources
 		Register(deprecatedEnvironmentClient).
 		Register(deprecatedTransactionsClient)
+
+	organizationsClient = resourcemanager.NewClient(
+		httpClient, cliLogger,
+		"organization", "organizations",
+		resourcemanager.WithTableConfig(resourcemanager.TableConfig{
+			Cells: []resourcemanager.TableCellConfig{
+				{Header: "ID", Path: "id"},
+				{Header: "NAME", Path: "name"},
+			},
+		}),
+		resourcemanager.WithListPath("elements"),
+	)
+
+	environmentClient = resourcemanager.NewClient(
+		httpClient, cliLogger,
+		"env", "environments",
```

Comment:
is `env` correct here?

Reply:
Yes, if we use envrionment it collides with the other resource 🥴